templates/CRM/Contact/Form/Search/Criteria/Basic.tpl

```diff
@@ -10,7 +10,7 @@
 <div class="advanced-search-fields basic-fields form-layout">
   {foreach from=$basicSearchFields item=fieldSpec}
     {assign var=field value=$form[$fieldSpec.name]}
-    {if $field}
+    {if $field && !in_array($fieldSpec.name, array('first_name', 'last_name'))}
```

Comment:
Do we suppress the fields here?

Reply:
@eileenmcnaughton no. As in sort_name.tpl we are already rendering these two fields,  reason why we are preventing first_name and last_name to be added as a separate field in advance search form.